eng/build.sh

```diff
@@ -360,6 +361,11 @@ while [[ $# > 0 ]]; do
       shift 2
       ;;
 
+     -usemonoruntime)
```

Comment:
The vmr option is [--use-mono-runtime](https://github.com/dotnet/dotnet/blob/ea11d078482522dfa30c5caae1a890a14f6ac909/build.sh#L12). Can you use the exact same name?

Reply:
Good point! The rest of the options in the runtime build.sh use a completely different convention, though (eg `outputrid`, `keepnativesymbols`). Whatever I go with, it will look out of place with the other convention :(

Since this option lives in build.sh, I tried to keep it consistent with the other options in build.sh.

I am happy to change it if it's okay with the runtime maintainers.

Reply:
Sounds reasonable to me

Reply:
I'm fine leaving out the dashes. I didn't realize you had done it to be consistent with the other arguments.

Reply:
I'm fine with the current state too.